TWLight/users/factories.py

```diff
@@ -29,7 +29,10 @@ class Meta:
         model = User
         django_get_or_create = ('username',)
 
-    username = 'alice'
+    # Multiple users with the same username can cause issues with group
```

Comment:
Please take advantage of factory boy's use of faker when you need random test data, like:
https://github.com/WikipediaLibrary/TWLight/blob/ae1ae76eea600d9ac747fe560de907d0e4f3ee2c/TWLight/resources/factories.py#L14